### PR TITLE
Release note 1.23.0

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit is a release note for 1.23.0.

Related: https://github.com/Samsung/ONE/issues/10802
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
This PR should be merged after #10809

File: docs/release/1.23/release-note-1.23.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release Note 1.23.0
+
+## ONE Compiler
+
+- Support more Op(s): GeLU
+- Support more option(s): `--fuse-gelu`
+- Support multiple backends compilation with a single configuration file
+- Upgrade Circle schema into 0.5
```
Comment (seanshpark) [inline docs/release/1.23/release-note-1.23.0.md:8]: `into` -> `to` ?

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**